yt_dlp/extractor/agora.py

```diff
@@ -165,7 +165,7 @@ def _real_extract(self, url):
         metadata = metadata[0]
 
         formats = []
-        for ext in ('aac', 'mp3'):
+        for ext in ('mp3',):
```

Comment:
If there are no other formats available, could you remove the for loop?

Reply:
Done